instrumentation/nacos-client-2.0.3/javaagent/src/test/java/com/alibaba/nacos/common/remote/client/RpcClientTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.alibaba.nacos.common.remote.client;
```

Comment:
usually we place test classes in the same package as the instrumentation

Reply:
I did this because `com.alibaba.nacos.common.remote.client.RpcClient#handleServerRequest` is a `protected` method. Is there a better way to handle this here?